packages/react/ds/src/header/header.tsx

```diff
@@ -185,6 +185,11 @@ export function Header({
               id="MobileMenuTrigger"
               type="checkbox"
             />
+            {tools?.menu?.label && (
+              <span className="gi-hidden sm:gi-block gi-text-2md gi-font-bold gi-text-white">
+                {tools?.menu?.label}
```

Comment:
Minor: If `tools?.menu?.label` is true then maybe replace it with `tools.menu.label`